README.md

```diff
@@ -1 +1,6 @@
-# basic-express-server
\ No newline at end of file
+# basic-express-server
+
```

Comment:
here a link to the [README template](https://github.com/codefellows/seattle-javascript-401n18/blob/main/ASSIGNMENT_TEMPLATE.md)